src/category_theory/adjunction.lean

```diff
@@ -29,6 +29,8 @@ structure adjunction (F : C ⥤ D) (G : D ⥤ C) :=
 (hom_equiv_unit' : Π {X Y f}, (hom_equiv X Y) f = (unit : _ ⟶ _).app X ≫ G.map f . obviously)
 (hom_equiv_counit' : Π {X Y g}, (hom_equiv X Y).symm g = F.map g ≫ counit.app Y . obviously)
 
+infix ` ⊣ `:15 := adjunction
```

Comment:
Good idea to introduce this notation. Could you also change the other examples of adjunctions that are out there, so that they also use this notations? Thanks!

Reply:
ok!

Reply:
Shall I also use `notation `𝟭` := functor.id` consistently throughout the category theory library?

Reply:
Hmm... I guess that's more work (even if it is trivial) and clutters up the PR. I'd rather have that in a separate PR.

Reply:
Also... maybe we should first think a bit about making that global notation...